dangermode/__init__.py

```diff
@@ -12,11 +12,13 @@ def activate_dangermode():
     Jupyter Notebook or JupyterLab.
     """
     import asyncio
-    import uvicorn
     import atexit
+
+    import uvicorn
+
     from dangermode.app import app
 
-    config = uvicorn.Config(app)
+    config = uvicorn.Config(app, host='0.0.0.0')
```

Comment:
We should pass this in as an argument so that in Docker we set the host to `0.0.0.0` but if anyone really runs this in full danger mode (on their local box) then it'll run only on localhost.